packages/volto/src/helpers/index.js

```diff
@@ -39,6 +39,7 @@ export {
   getLanguageIndependentFields,
 } from '@plone/volto/helpers/Content/Content';
 export {
+  _applyBlockInitialValue,
```

Comment:
@wesleybl Usually when you have a method that starts with _ it's implied that it is private.
I don't think it should thus be exported for helpers, if that is the case that it would be useful
for add-ons to call this helper then I would remove the _.
Ex info
https://css-tricks.com/implementing-private-variables-in-javascript/#aa-using-an-underscore
but if you google js private convention you will see that this was very popular before true private
fields

Reply:
@ichim-david done.